brainstormforce/ultimate-addons-for-gutenberg
# UAG-90 Improvement - Static CSS size optimization

### Description

Trello - https://trello.com/c/hDLQyYkZ/264-static-css-size-optimization

### Screenshots


### Types of changes




- non-breaking change CSS Improvement

### How has this been tested?

- Check Social Share, Team, and Testimonial.
- Drag and drop the block and check every setting in which layout should be change and block should not be broken.

### Checklist:
